ante/fee.go

```diff
@@ -85,3 +70,24 @@ func isOracleTx(msgs []sdk.Msg) bool {
 
 	return true
 }
+
+// getTxPriority returns naive tx priority based on the fee amount and oracle tx check.
+func getTxPriority(fee sdk.Coins, isOracle bool) int64 {
+	var priority int64
+	for _, c := range fee {
```

Comment:
```suggestion
	// sets priority to the lowest positive Amount in fee Coins (regardless of Denom)
	for _, c := range fee {
```
Just a comment so it's easier to understand what the placeholder is doing

Reply:
this will be changed anyway. I will update the function comment